conf/upload_supporting_evidence.routes

```diff
@@ -12,6 +12,7 @@ POST   /supporting-evidence/upload-supporting-evidence-virus-check          uk.g
 GET    /supporting-documents/delete-supporting-evidence/:id                 uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.supportingdocs.SupportingEvidenceController.deleteSupportingEvidence(id: uk.gov.hmrc.cgtpropertydisposalsfrontend.models.upscan.UploadReference, addNew: Boolean)
 
 GET    /supporting-documents/expired                                        uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.supportingdocs.SupportingEvidenceController.supportingEvidenceExpired()
```

Comment:
whilst you're here - could you substitute `supporting-documents` with `supporting-evidence` please

Reply:
whilst we don't have the proper url's from Jean I'd leave it as it is now since it doesn't break anything.

Reply:
I'll ask for the proper url for this page though

Reply:
the original tickets for all this piece of work had it as `support-evidence` as designed by Jean

Reply:
ah ok - I'll change it then

Reply:
cleaned up the routes file now